update guide for configuring windows on VMs - KFT0022321 (#7104)
ovh/docs: pages/bare_metal_cloud/dedicated_servers/activate_windows_vm_hyperv/guide.en-asia.md

This tutorial assumes that you have already installed the Hyper-V role and have access to the Hyper-V Manager. If you have not done this, you can refer to Microsoft's guide to installing the Hyper-V role [here](https://docs.microsoft.com/en-us/windows-server/virtualization/hyper-v/get-started/install-the-hyper-v-role-on-windows-server){.external}.

> The steps below are for standard activation only. If you have a Windows Datacenter licence, you can also proceed with activation via [AVMA activation](https://learn.microsoft.com/en-us/windows-server/get-started/automatic-vm-activation?tabs=server2019#system-requirements){.external}, which we recommend.

Reviewer: ```suggestion
The steps below are for standard activation only. If you have a Windows Datacenter licence, we recommend proceeding with [Automatic Virtual Machine Activation (AVMA)](https://learn.microsoft.com/en-us/windows-server/get-started/automatic-vm-activation?tabs=server2019#system-requirements){.external}.
```
